.github/workflows/linux.yml

```diff
@@ -362,3 +366,51 @@ jobs:
         rm -rf build-install
         ./vendored-meson/meson/meson.py install -C build --destdir ../build-install --tags=runtime,python-runtime,devel
         python tools/check_installed_files.py $(find ./build-install -path '*/site-packages/numpy') --no-tests
+
+
+
+  Linux_Python_311_32bit_full_with_asserts:
+    name: i686, cp311, full, with asserts
+    needs: [smoke_test]
+    runs-on: ubuntu-latest
+    container:
+       # There are few options for i686 images at https://quay.io/organization/pypa,
+       # use the glibc2.17 one (manylinux2014)
```

Comment:
It doesn't matter much, but we did drop `manylinux2014` support already, so if there's a `2_28` image, that may be slightly better.